server/src/main/java/org/opensearch/search/internal/SearchContext.java

```diff
@@ -97,6 +97,7 @@ public abstract class SearchContext implements Releasable {
     private final List<Releasable> releasables = new CopyOnWriteArrayList<>();
     private final AtomicBoolean closed = new AtomicBoolean(false);
     private InnerHitsContext innerHitsContext;
+    protected boolean concurrentSegmentSearchEnabled;
```

Comment:
Please remove `concurrentSegmentSearchEnabled`

Reply:
The reason why I have it still is so that `setConcurrentSegmentSearchEnabled` can be used by the tests. I can just move that into `TestSearchContext` though. Thoughts?

Reply:
Yep, `TestSearchContext` it is :-)